larpandoracontent/LArCheating/CheatingPfoCreationAlgorithm.cc

```diff
@@ -129,6 +130,27 @@ void CheatingPfoCreationAlgorithm::CreatePfos(const MCParticleToClusterListMap &
 
             const ParticleFlowObject *pPfo(nullptr);
             PANDORA_THROW_RESULT_IF(STATUS_CODE_SUCCESS, !=, PandoraContentApi::ParticleFlowObject::Create(*this, pfoParameters, pPfo));
+
+            if (m_addVertices)
+            {
+                const VertexList *pVertexList = NULL; std::string vertexListName;
+                PANDORA_THROW_RESULT_IF(STATUS_CODE_SUCCESS, !=, PandoraContentApi::CreateTemporaryListAndSetCurrent(*this, pVertexList, vertexListName));
```

Comment:
Probably better to just create the temporary list once, as for the Pfos, above. Then fill it as we build Pfos and save it at the end.